### ISSUE TITLE
[core] 3.3.3 Does not build with blitz.js

### ISSUE THREAD

**Issue by JohannesKlauss** [mantinedev/mantine#487]
The CI pipeline for my project fails when trying to update the core package to 3.3.3.

It fails with:

```
./node_modules/@mantine/styles/src/theme/ColorSchemeProvider.tsx:9:67
Type error: Argument of type 'null' is not assignable to parameter of type 'ColorSchemeContextProps'.

   7 | }
   8 | 
>  9 | const ColorSchemeContext = createContext<ColorSchemeContextProps>(null);
     |                                                                   ^
  10 | 
  11 | export function useMantineColorScheme() {
  12 |   const ctx = useContext(ColorSchemeContext);
```

This is my tsconfig.json

```
{
  "compilerOptions": {
    "target": "es5",
    "lib": ["dom", "dom.iterable", "esnext"],
    "baseUrl": "./",
    "allowJs": true,
    "skipLibCheck": true,
    "strict": false,
    "strictNullChecks": true,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "noUncheckedIndexedAccess": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "incremental": true,
    "tsBuildInfoFile": ".tsbuildinfo",
    "experimentalDecorators": true
  },
  "exclude": ["node_modules", "**/*.e2e.ts", "cypress"],
  "include": ["blitz-env.d.ts", "**/*.ts", "**/*.tsx"]
}
```

And I am using Typescript 4.5.2

It is working with core version 3.0.5, but after that it fails for every version.

**Comment by rtivital** [comment]
It seems like you have skipLibCheck set to false on CI. Can you replicate this issue while typecheck locally?
```
tsc --noEmit
```

**Comment by JohannesKlauss** [comment]
Hmh... I don't think that the CI sets `skipLibCheck` to `false`. The CI uses the same tsconfig.

I am able to replicate this by trying to build the application locally, yes.
And you are probably right with the styles package. Running `tsc --noEmit` produces 41 errors, all of them in the styles package.

I just saw the first error being the ColorSchemeContext and then the build process exited.

**Comment by rtivital** [comment]
Okay, thanks for checking, I've published 3.3.4 which excludes src folders from all packages, can you update and see if it resolves your issue?

**Comment by JohannesKlauss** [comment]
@rtivital It indeed fixed the bug. Thank you so much for the extremely quick fix!
